src/popup/components/LatestTransactionsCard.vue

```diff
@@ -82,6 +84,7 @@ export default defineComponent({
             (transaction: ITransaction) => ({
               ...transaction,
               transactionOwner: address,
+              direction: getTxDirection.value(transaction.tx, address),
```

Comment:
Usage of `getTxDirection` is incorrect here, transactions may have innerTx inside. I would suggest to use `direction` from `transactionTx` composable.